exercises/practice/affine-cipher/.docs/instructions.append.md

```diff
@@ -0,0 +1,35 @@
+# Hints
+
+You need to implement the `decode` and `encode` functions, which decode and encode a `String` using an Affine cipher.
+You can use the provided signature if you are unsure about the types, but don't let it restrict your creativity.
+
+This exercise works with textual data.
+For historical reasons, Haskell's `String` type is synonymous with `[Char]`, a list of characters.
+For more efficient handling of textual data, the `Text` type can be used.
+
+As an optional extension to this exercise, you can
+
+- add `- text` to your list of dependencies in package.yaml.
+- import `Data.Text` in [the following
+  way](https://hackernoon.com/4-steps-to-a-better-imports-list-in-haskell-43a3d868273c):
+
+```haskell
+import qualified Data.Text as T
+import           Data.Text (Text)
+```
+
+- use the `Text` type e.g. `decode :: Text -> Text` and refer to
+  `Data.Text` combinators as e.g. `T.pack`.
```

Comment:
`Text.pack` is unnecessary when `OverloadedStrings` is enabled. Do we want to recommend this extension? Though it is idiomatic, it is one more complication as well.

`OverloadedStrings` has been mentioned once before on this track: in [bob/.approaches/text/content.md](https://github.com/exercism/haskell/blob/000e69d1d654691503ebf0e2f1c11664a5d371b6/exercises/practice/bob/.approaches/text/content.md?plain=1#L44-L82).